paasta_tools/cli/schemas/tron_schema.json

```diff
@@ -139,6 +140,15 @@
                 },
                 "trigger_timeout": {
                     "$ref": "#definitions/time_delta"
+                },
+                "spark_cluster": {
```

Comment:
Do you think this is better than `spark_paasta_cluster`? 
Same with `spark_pool`

Reply:
I don't really have a problem with it, but we can change it if you want.